docs/_includes/example.njk

```diff
@@ -70,7 +70,9 @@
 {% if figmaLink %}
   <div class="app-tabs__panel govuk-!-padding-2 app-tabs__panel--hidden" id="figma-default-{{ id }}" role="tabpanel" aria-labelledby="tab_figma-default-{{ id }}">
 
-To use the component in your design, add the MoJ Figma Kit to your Figma libraries and find the component using the Assets tab.
+This component is in the ‘Assets’ tab in the MoJ Figma Kit.
+
+If you’re external to MoJ, you can download the Kit and add it as a library to your Figma files. Updates to the Kit are only shown automatically for Figma users at MoJ, though, so you’ll need to add it again.
```

Comment:
I think the last part of this sentence reads a bit clunkily.

"_... though, so you'll need to add it again._"

Reply:
OK. I'll look at this again.